pkg/compact/compact.go

```diff
@@ -470,6 +481,98 @@ func (cg *Group) Resolution() int64 {
 	return cg.resolution
 }
 
+// metrics related to planning and compaction progress
+type ProgressMetrics struct {
+	NumberOfIterations    *prometheus.GaugeVec
+	NumberOfBlocksToMerge *prometheus.GaugeVec
+	// figure out where to add groupKey labels to this
```

Comment:
Let's remove this line as it is done.

Reply:
Thanks!
I'll remove the other comments too once the PR LGTM.